docker-compose.yml

```diff
@@ -2,6 +2,16 @@ version: '2'
 
 # docker-compose file to launch jenkins_master/ELK stack instead of VM.
 services:
+  proxy:
+    image: traefik
+    command: --web --docker --docker.domain=${PROXY_DOMAIN} --logLevel=${PROXY_LOG_LEVEL}
+    ports:
+      - "80:80"
+      - "8080:8080"
+      - "443:443"
```

Comment:
Looks like there's a conflicting 443 allocation for Kibana later in the file (I wouldn't've found it, but, Travis did!)